core/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -397,15 +414,17 @@ public static InputTableConfig getInputTableConfig(JobConf job, String tableName
      *          the table name for which the scanner is configured
      * @since 1.6.0
      */
-    protected abstract void setupIterators(JobConf job, Scanner scanner, String tableName, RangeInputSplit split);
+    @Deprecated
```

Comment:
When deprecated, in addition to adding the annotation we usually add something like `@deprecated since 1.7.0` to the javadoc also.   That documents when it was deprecated, I don't think it really serves any other purpose since we have the annotation.